client/Components/GuestDropdown.jsx

```diff
@@ -1,11 +1,21 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import styles from './GuestDropdown.css';
-import GuestItem from './GuestItem.jsx';
+import GuestItem from './GuestItem';
+
+// const mapStateToProps = (state) => {
+//   return {
+//     checkin: state.checkin,
+//     checkout: state.checkout,
+//   };
+// };
 
 class GuestDropdown extends React.Component {
   constructor(props) {
     super(props);
+    const { maxGuests } = this.props;
```

Comment:
Are you getting the data you needed from maxGuests without referencing this.props.data like you did on line 28? Form is currently only passing duration and data props.

Reply:
maxGuests is from inside props.data; I just hadn't cleaned up the constructor.

I have now!